Makefile

```diff
@@ -510,6 +510,18 @@ out/doc/api/%.html: doc/api/%.md
 		fi
 	[ -x $(NODE) ] && $(NODE) $(gen-html) || node $(gen-html)
 
+# check if ./node is actually set, else use user pre-installed binary
+gen-man = tools/doc/generate.js --format=man $< > $@
+out/doc/api/%.3: doc/api/%.md
+	@[ -e tools/doc/node_modules/js-yaml/package.json ] || \
+		[ -e tools/eslint/node_modules/js-yaml/package.json ] || \
+		if [ -x $(NODE) ]; then \
+			cd tools/doc && ../../$(NODE) ../../$(NPM) install; \
+		else \
+			cd tools/doc && node ../../$(NPM) install; \
```

Comment:
Do we actually need `npm install` since `node_modules` is vendored?

Reply:
would definitely prefer *not* having `npm install` here.